cluster-autoscaler/cloudprovider/azure/test/templates/cluster-template-prow-aks-aso-cluster-autoscaler.yaml

```diff
@@ -49,6 +49,10 @@ spec:
           enabled: true
       servicePrincipalProfile:
         clientId: msi
+      sku:
+        name: Base
+        tier: ${AKS_TIER:=Standard}
+      supportPlan: ${AKS_SUPPORT_PLAN:=KubernetesOfficial}
```

Comment:
These are the current defaults, though we could make `tier: Premium` and `supportPlan: AKSLongTermSupport` the defaults (or even hardcode them) here. That would cost more money, but we wouldn't need to manage which versions require LTS and which don't as that set evolves over time. Only preview versions (currently 1.34.0) can't be created with AKSLongTermSupport, but even newer Kubernetes versions (1.31-1.33) can.

Reply:
https://github.com/kubernetes/test-infra/pull/35826 should be back-compat (so not breaking other PRs in flight against master) but enable us to test this change here before we merge